jobs/dynatrace-oneagent-windows/templates/pre-start.ps1.erb

```diff
@@ -110,7 +110,14 @@ function downloadAgent() {
 
 		Try {
 			installLog "INFO" "Downloading Dynatrace agent from $cfgDownloadUrl to $installerFile"
-			Invoke-WebRequest $cfgDownloadUrl -Outfile $installerFile -UserAgent $userAgent
+			
+			if($cfgProxy -ne "") {
+				installLog "INFO" "Proxy settings found, setting system proxy to $cfgProxy"
```

Comment:
That log message actually isn't true.
We're not configuring the system settings here, just downloading via proxy.